quickwit/quickwit-serve/src/index_api/rest_handler.rs

```diff
@@ -263,12 +271,22 @@ pub struct ListSplitsQueryParams {
     pub end_create_timestamp: Option<i64>,
 }
 
+#[derive(Serialize, Deserialize, Debug, utoipa::ToSchema)]
+pub struct ListSplitsResponse {
+    #[serde(default)]
+    pub offset: usize,
+    #[serde(default)]
+    pub size: usize,
+    #[serde(default)]
+    pub splits: Vec<Split>,
```

Comment:
We will probably use pagination for other resources: indexes for example. Replacing the field `splits` would be better.

Reply:
Currently it's only used as the response of list split operation, that's why I use splits as the name. Maybe we can define different structures to to be used in different responses